spec/unit/knife/bootstrap/client_builder_spec.rb

```diff
@@ -41,6 +41,24 @@
     client_builder
   }
 
+  context "#run" do
+    let(:client) { Chef::ApiClient.new }
+    let(:node) { Chef::Node.new }
+
+    before do
+      # mock out the rest of #run
+      expect(client_builder).to receive(:sanity_check)
+      expect(client_builder).to receive(:create_client!).and_return(client)
+      expect(client_builder).to receive(:create_node!).and_return(node)
+    end
+
+    it "returns the created client and nodeeturns " do
```

Comment:
was sorta hoping nodeeturns would catch on. but yeah I dont think its gonna stick. People seem attached to "node"